capnpc-go/capnpc-go.go

```diff
@@ -410,6 +409,14 @@ func (g *generator) Value(rel *node, t schema.Type, v schema.Value) (string, err
 	}
 }
 
+func isAnyCap(ap schema.Type_anyPointer) bool {
+	unconstrained := ap.
+		Which() == schema.Type_anyPointer_Which_unconstrained
+	capability := ap.Unconstrained().
```

Comment:
Is it ok to do this without first verifying `unconstrained == true`? My memory is this should panic if it's another variant, but I could be misremembering.

Reply:
It should be safe; we were doing it in an equivalent way before.  I believe union types will panic, but this is a group.